### DISCUSSION TITLE
Export allure results to email

### DISCUSSION THREAD

**Discussion by Lnoy** [allure-framework/allure2#1487] [General Discussion]
Hi there,

i am pretty much new to allure and would like some guidance in the next issue:
I have a project with nightly runs via argo and integrate with allure. i can see the reports in allure system once run is finish.
My question is: 
i want to add a job that automatically send an email of allure results, it can be a screenshot of the diagrams and numbers. where do i need to add it? in argo? or in my project?
i use kotlin and gradle in my project.

Thanks,
Noy.

**Comment by cheshi-mantu** [comment]
Hi,
try this one: https://github.com/qa-guru/allure-notifications
